.gitignore

```diff
@@ -83,5 +45,6 @@ data/isins_missing_bbg_data.csv
 data/fund_isins_without_bbg_data.csv
 
 # Mac files
-working_dir/DS_STORE
+.DS_STORE
+**/.DS_STORE
```

Comment:
I think .gitignore is case sensitive. Did you mean this?

```
.DS_Store
**/.DS_Store
```

Reply:
woops, you are absolutely right.